docs/file-specification.md

```diff
@@ -48,7 +48,9 @@ Byte encoding is little-endian.
 |72-75 |uint32 |Encoded version number
 |76-79 |uint32 |Encoded source format 0=SEG-Y, 10=ZGY, 20=numpy
 |80-83 |uint32 |Encoded header-detection method ***
-|84-959 |---  | --- Unused ---
+|84-92   |float64  | Minimum sample time/depth*****
```

Comment:
Here you want to use 91: 84-91

Reply:
Yes, looks like this is the interval format here...